static/app/types/organization.tsx

```diff
@@ -70,6 +70,7 @@ export interface Organization extends OrganizationSummary {
     projectLimit: number | null;
   };
   relayPiiConfig: string | null;
+  requiresSso: boolean;
```

Comment:
if we merge BE and FE at the same time, this is not guaranteed to be sent from the backend when the frontend is merged, especially since backend deploys take longer. it might error unless we also allow `null` here

Reply:
Would it be more sensible to do `requiresSso?`. I see no problem treating `null` and `false` interchangeably.

Reply:
i think that works too

Reply:
Let's just deploy the backend first.